dspace/config/modules/versioning.cfg

```diff
@@ -16,3 +20,7 @@ versioning.item.history.include.submitter=false
 # If you want to allow submitters to create new versions of there items, set
 # the property submitterCanCreateNewVersion true.
 # versioning.submitterCanCreateNewVersion=false
+
+# The property versioning.block.entity is used to disable versioning
+# for items with EntityType, the default value is true if it unset.
+# versioning.block.entity=
```

Comment:
Tiny thing, usually we put the default value after the `=` sign.  So, we might want to make this say `# versioning.block.entity=true`